src/features/pastelStatistics/components/card/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import bg1 from '../../assets/images/bg1.png'
+import bg2 from '../../assets/images/bg2.png'
+import bg3 from '../../assets/images/bg3.png'
+import bg4 from '../../assets/images/bg4.png'
+import bg5 from '../../assets/images/bg5.png'
+import bg6 from '../../assets/images/bg6.png'
+import bg7 from '../../assets/images/bg7.png'
+import bg8 from '../../assets/images/bg8.png'
+import bg9 from '../../assets/images/bg9.png'
```

Comment:
Can we move this images to the `pastelChartFields` object? This way, they would be associated with actual routes, and `CardProps` can have property to pass an image.